src/lib/contact-list.js

```diff
@@ -25,7 +26,8 @@ function getPreferredContact (contacts, rolePriority) {
     if (acc) {
       return acc
     }
-    return find(contacts, { role })
+
+    return contacts.find(o => o.role === role)
```

Comment:
I think we usually put brackets around single parameters so this and others would need to be changed.
```suggestion
    return contacts.find((o) => o.role === role)
```